public/javascripts/script.js

```diff
@@ -8,24 +8,41 @@ for (let i = 0; i < mashPotatoes.length; i++) {
 
 // Iteration 1 using callbacks
 addFood(steak[0], '#steak', () => {
-  // ... your code here
-  addFood(steak[1], '#steak', () => {
-
-  })
+ addFood(steak[1], '#steak', () => {
+   addFood(steak[2], '#steak', () => {
+    addFood(steak[3], '#steak', () => {
+     addFood(steak[4], '#steak', () => {
+      addFood(steak[5], '#steak', () => {
+       addFood(steak[6], '#steak', () => {
+        addFood(steak[7], '#steak', () => {
+           });
+          });
+         });
+       });
+      });
+    });
+  });
 });
-
-
-
 // Iteration 2 using `.then()`
 addFood(mashPotatoes[0], '#mashPotatoes').then(() => {
-  // ... your code here
-  addFood(mashPotatoes[1], '#mashPotatoes')
+  addFood(mashPotatoes[1], '#mashPotatoes').then(() =>{
+    addFood(mashPotatoes[2], '#mashPotatoes').then(() =>{
+      addFood(mashPotatoes[3], '#mashPotatoes').then(() =>{
+        addFood(mashPotatoes[4], '#mashPotatoes').then(() =>{
+        });
+      });
+    });
+  });
 });
-
 // Iteration 3 using async and await
-
-  async function makeFood(step) {
-    // ... your code here
-    
+  
+async function makeFood(steps, id) {
+  for (let i = 0; i < steps; i++) {
+    await addFood(brusselSprouts[i], id);
   }
   makeFood(eachStep);
```

Comment:
vc pode remover o `makeFood(eachStep)` que vai dar um erro pois o "eachStep" não existe.
Não aparece o erro na hora pq ele acontece depois do `for` q é a ultima coisa a carregar os comandos do código.

E no caso vc pode colocar o carregamento da img aqui dentro tb.